x/stakeibc/client/cli/tx_claim_undelegated_tokens.go

```diff
@@ -13,17 +13,24 @@ import (
 
 var _ = strconv.Itoa(0)
 
+// string creator = 1;
+//   string hostZoneId = 2;
+//   uint64 epoch = 3;
+//   string sender = 4;
+// }
+
 func CmdClaimUndelegatedTokens() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "claim-undelegated-tokens [host-zone] [max-claims]",
+		Use:   "claim-undelegated-tokens [host-zone] [epoch] [sender]",
```

Comment:
should we parse the sender from the args? seems odd to allow the user calling this function to choose the sender arbitrarily. you can get the user with `clientCtx.GetFromAddress().String()` I believe, as in line 41